.github/workflows/production.yml

```diff
@@ -0,0 +1,195 @@
+name: Production
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  FIREBASE_PROJECT_ID: mm-static-site-production
+  FIREBASE_SERVICE_ACCOUNT: "${{ secrets.PRODUCTION_FIREBASE_SERVICE_ACCOUNT }}"
+
+jobs:
+  cache-keys:
+    name: Cache keys
+    runs-on: ubuntu-latest
+
+    outputs:
+      build: build-${{ steps.build.outputs.hash }}-v1
+      node-modules: node-modules-${{ steps.node-modules.outputs.hash }}-v1
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+
+      - id: build
+        run: echo "::set-output name=hash::${{ hashFiles('content/**', 'firebase/**', 'functions/src/**', 'src/**', 'static/**', '**/package.json', '**/yarn.lock') }}"
```

Comment:
This should include `bin/**`, `sitemap-query-string.js` should be moved there, and the script in `package.json` that runs it should be updated